docs/source/getting-started/installation.rst

```diff
@@ -27,14 +35,14 @@ To install the SDK on Mac or Linux:
 1. Run::
 
      curl -sSL https://get.daml.com/ | sh
-2. If prompted, add ``~/.daml/bin`` to your PATH.
 
-   If you don't know how to do this, try following `these instructions for MacOS <https://hathaway.cc/2008/06/how-to-edit-your-path-environment-variables-on-mac/>`_ or `these instructions for Windows <https://www.java.com/en/download/help/path.xml>`_.
+2. Add ``~/.daml/bin`` to your PATH. You can find the instructions here on how to do it on `MacOS <https://hathaway.cc/2008/06/how-to-edit-your-path-environment-variables-on-mac/>`_ and `Linux <https://stackabuse.com/how-to-permanently-set-path-in-linux/>`_.
```

Comment:
Even more links to very unofficial docs/blogs/… :confused:

Reply:
@cocreature Added our own instructions now. If you have the time please review 😄